### PR TITLE
Grdddj/screenshots improvements

### PR DESCRIPTION
**Pull request by grdddj**
Loosely connected with https://github.com/trezor/trezor-user-env/issues/157:
- saving all the screenshots from emulators, when users want - each emulator session has its own directory
- mounting a docker volume to enable saving screenshots from the running container
- documentation and explanation in dashboard

### CODE REVIEW

**Comment by grdddj** [comment]
I felt it could be useful to have that automatic way to delete the red square - and storing it in repo means we will not lose it

**Comment by grdddj** [comment]
Rebased on `master` and updating python dependencies

File: pyproject.toml [hunk 1]
```diff
@@ -10,6 +10,7 @@ termcolor = "^1.1.0"
 trezor = "^0.13.0"
 websockets = "^10.1"
 psutil = "^5.8.0"
+Pillow = "^9.1.0"
```
Comment (tsusanka) [inline pyproject.toml:13]: Does it need to be capitalized?

**Comment by grdddj** [comment]
> Just some nit-picks. Could you also make a screenshot of the dashboard? Just wondering what it looks like now.

What changed are just the dots-underlined checkboxes, and when you hover over them, you will see some additional details about what that checkbox does

image

File: pyproject.toml (hunk 1 above)
Comment (grdddj) [reply pyproject.toml:13]: I think it is a capital by default. And it is the same in `firmware` - https://github.com/trezor/trezor-firmware/blob/master/pyproject.toml#L57

File: src/helpers.py [hunk 2]
```diff
@@ -25,3 +26,31 @@
 def log(text: str, color: str = "blue") -> None:
     logging.info(text)
     print(colored(text, color))
+
+
+def replace_red_square(screenshot_location: str) -> None:
```
[the remark this answers is not in the document]
Comment (grdddj) [reply src/helpers.py]: Sure, changed in [31eaac9](https://github.com/trezor/trezor-user-env/pull/162/commits/31eaac99b3904f31054d602d1c66768637fdd4fe)

**Review by tsusanka — APPROVED** [review]
LGTM!

**Pull request merged**